Cargo.toml

```
@@ -5,14 +5,18 @@ homepage = "https://github.com/debris/tiny-keccak"
 documentation = "https://docs.rs/tiny-keccak/"
 repository = "https://github.com/debris/tiny-keccak"
 license = "CC0-1.0"
-name = "tiny-keccak"
+name = "tari_tiny-keccak"
```

```suggestion
name = "tari-tiny-keccak"
```
or 
`tari_tiny_keccak`